### ISSUE TITLE
Arrtibute name / property / itemprop on element meta / link not valid at this point.

### ISSUE THREAD

**Issue by Meteor0id** [validator/validator#675]
Error: Attribute name not allowed on element meta at this point.
`<meta itemprop="foundingDate" name="foundingDate" content="2017-12-22" />`

Error: Attribute property not allowed on element meta at this point.
`<meta itemprop="email" name="contact" property="business:contact_data:email" content="vraagofopmerking@ffsociety.nl" />`

etc. etc.

I am sure many of you are aware of this issue but let's make work of it so beginners are not discurraged by false errors when properly implementing microdata and RFDa.

**Comment by sideshowbarker** [comment]
```
<meta itemprop="foundingDate" name="foundingDate" content="2017-12-22" />
```

The message from the checker is correct. That element isn’t valid. The reason it’s not valid is this:

https://html.spec.whatwg.org/multipage/semantics.html#the-meta-element:attr-meta-name-4

> Exactly one of the `name`, `http-equiv`, `charset`, and `itemprop` attributes must be specified.

The element in the question has both the `itemprop` and `name` attributes. So it’s not valid, because the HTML spec says that it must instead have exactly one of them — either the `itemprop` attribute OR the `name` attribute. I can’t have both.

```
<meta itemprop="email" name="contact" property="business:contact_data:email" content="vraagofopmerking@ffsociety.nl" />
```
That element also isn’t valid — for the same reason that the first element is in the question isn’t valid. 

If you want to use the `property` attribute on that element, then drop the `itemprop` attribute. Don’t attempt to use both microdata and RDF attributes on the same element.

**Comment by sideshowbarker** [comment]
Closing per https://github.com/validator/validator/issues/675#issuecomment-401963309 but feel free to comment further here and I’ll respond if you do comment

**Comment by tc2u4me** [comment]
Attribute "name" not allowed on element "meta" at this point.

<meta name=viewport content="width=device-width, initial-scale=1">

Thanks for keeping this open. The more I read the more lost I get. Everything I read says to drop "name" I think I understand as. If I drop name, what goes in it's place? My problem is 25% of visitors get the wrong resolution. Can't figure it out.

**Comment by sideshowbarker** [comment]
@tc2u4me `<meta name=viewport content="width=device-width, initial-scale=1">` is not an error and the checker does not report it as an error.
